db/plan.sh

```diff
@@ -25,6 +27,14 @@ pkg_bin_dirs=(bin)
 pkg_include_dirs=(include)
 pkg_lib_dirs=(lib)
 
+do_prepare() {
+  # Many thanks to Arch Linux
+  # https://git.archlinux.org/svntogit/packages.git/plain/trunk/atomic.patch?h=packages/db
+  pwd
+  ls $PLAN_CONTEXT/patches/atomic.patch -la
```

Comment:
do we need the `ls` ?

Reply:
Nope, pre-coffee-me committed his debugging.